core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -142,7 +147,10 @@ class DynamicConfigManager(private val zkUtils: ZkUtils,
       val fullSanitizedEntityName = entityPath.substring(index + 1)
 
       val entityConfig = AdminUtils.fetchEntityConfig(zkUtils, rootEntityType, fullSanitizedEntityName)
-      logger.info(s"Processing override for entityPath: $entityPath with config: $entityConfig")
+      val loggableConfig = entityConfig.asScala.map {
```

Comment:
Also, is there any test that exercises this class with Scram enabled so that I can verify the fix?